util/docker/open3d-xvfb/setup/headless_sample.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Open3D: www.open3d.org
+# The MIT License (MIT)
+# See license file or visit www.open3d.org for details
+
+import sys, os
+sys.path.append("../..")
+from py3d import *
+import numpy as np
+import matplotlib.pyplot as plt
+
+import faulthandler
+faulthandler.enable()
+
+def custom_draw_geometry_with_camera_trajectory(pcd):
```

Comment:
I am also made Advanced/headless_rendering.py. Can we make it just one script, or drop this? It is easy to say the docker is mainly used with vnc connection.

Reply:
DONE